indico/modules/events/registration/blueprint.py

```diff
@@ -47,6 +54,25 @@
                  'close_regform', RHRegistrationFormClose, methods=('POST',))
 _bp.add_url_rule('/manage/registration/<int:reg_form_id>/schedule',
                  'schedule_regform', RHRegistrationFormSchedule, methods=('GET', 'POST'))
+_bp.add_url_rule('/manage/registration/<int:reg_form_id>/modify/', 'modify_regform', RHModifyRegForm)
+
+# Sections
+_bp.add_url_rule('/manage/registration/<int:reg_form_id>/modify/sections', 'add_section', RHRegFormAddSection,
+                 methods=('POST',))
+_bp.add_url_rule('/manage/registration/<int:reg_form_id>/modify/sections/<section_id>', 'modify_section',
+                 RHRegFormModifySection, methods=('PATCH',))
```

Comment:
fyi, this could be one RH for both DELETE and PATCH and then you'd use `_process_PATCH` and `_process_DELETE` to the two actions